.github/actions/build-binaries/action.yaml

```
@@ -58,6 +61,8 @@ runs:
         cd dist/algokit/
         if [ "${{ inputs.production_release }}" == "true" ]; then
           tar -zcvf ../../algokit-${{ env.RELEASE_VERSION }}-${{ inputs.operating_system }}-${{ inputs.architecture }}-py${{ inputs.python_version }}.tar.gz *
+          echo ${{ inputs.distribution }} > ./_internal/distribution_channel/distribution_channel.txt
```

We should probably use a similar conventions to the `last-version-check` file. So something like below. This also allows for extensibility if we want to write other package configuration.

```suggestion
          echo ${{ inputs.distribution }} > ./_internal/algokit/distribution-method
```